DependencyInjection/HttplugExtension.php

```diff
@@ -88,6 +89,11 @@ protected function configureClients(ContainerBuilder $container, array $config)
         // Alias the first client to httplug.client.default
         if ($first !== null) {
             $container->setAlias('httplug.client.default', 'httplug.client.'.$first);
+        } elseif (isset($config['_inject_collector_plugin'])) {
+            // No client was configured. Make sure to inject history plugin to the auto discovery client.
+            $container->register('httplug.client', PluginClient::class)
```

Comment:
was `httplug.client` not available at all before? or did this happen somewhere else? i am a bit confused by this.